File: DataSafe/app/src/main/java/me/aheadlcx/datasafe/Person.java
package me.aheadlcx.datasafe;

import android.support.annotation.NonNull;

/**
 * Description:
 * author: aheadlcx
 * Date:2018/3/27 上午11:37
 */

public class Person implements Comparable<String>{
    public String name;

    public Person(int name) {
        this.name = name + "";
    }

    @Override
    public int compareTo(@NonNull String o) {
            return 0;
    }
}

File: App311/app/src/main/java/me/aheadlcx/app311/Person.kt
package me.aheadlcx.app311

/**
 * Description:
 * author: aheadlcx
 * Date:2018/4/8 下午12:19
 */
data class Person(val name: String = "aheadlcx", val email: String = "aheadlcxzhang@gmail", val
age: Int = 0) {
    var number: String = "1"
    fun show(): String {
        return this.name + this.email + this.age;
    }
}
